homeassistant/components/climate/homematic.py

```diff
@@ -101,12 +97,15 @@ def current_operation(self):
     @property
     def operation_list(self):
         """Return the list of available operation modes."""
-        op_list = []
+        # HMIP use set_point_mode for operation
+        if HMIP_CONTROL_MODE in self._data:
+            return [STATE_MANUAL, STATE_AUTO, STATE_BOOST]
```

Comment:
`STATE_BOOST` isn't part of climate base component, so we're not allowed to use that for operation mode.

Also, the platform is redefining `STATE_MANUAL` etc, instead of importing these from the climate component.

Reply:
That is correct. I let it stay like it is because I refactor the hole climate on my new work next year.

Reply:
Ok. I would like to see a bit longer description in this PR on background and scope. Why do we do these changes? We would expect that from a contributor.